api/v1alpha1/clienttrafficpolicy_types.go

```diff
@@ -93,6 +93,10 @@ type HTTP1Settings struct {
 	// EnableTrailers defines if HTTP/1 trailers should be proxied by Envoy.
 	// +optional
 	EnableTrailers *bool `json:"enableTrailers,omitempty"`
+	// PreserveHeaderCase defines if Envoy should preserve the letter case of headers.
+	// By default, Envoy will lowercase all the headers.
+	// +optional
+	PreserveHeaderCase *bool `json:"preserveHeaderCase,omitempty"`
```

Comment:
I really don't encourage this API, because headers should not be case sensitive.

Reply:
fine, let us knee to real world.